### DISCUSSION TITLE
Network Layer 공부 (1)

### DISCUSSION THREAD

**Discussion by KwonNaeHyeon** [BDD-CLUB/challenge-every-week-posting#233] [3. CS(컴퓨터 공학) 관련 글 모음]
[Network Layer(1)](https://lateral-value-9a9.notion.site/10-67e2f6bf95ca4df7a9afe6c537ceffc9?pvs=4)
이번에 과제를 하기 전에 네트워크 레이어에서 배운 전반적인 내용을 정리해보았습니다.

**Comment by shkisme** [comment]
헐 ㄷㄷ 학습의 깊이가 장난아니네요... 멋집니다 👍
